src/util/bitvector_expr.h

```diff
@@ -0,0 +1,676 @@
+/*******************************************************************\
+
+Module: API to expression classes for bitvectors
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_UTIL_BITVECTOR_EXPR_H
+#define CPROVER_UTIL_BITVECTOR_EXPR_H
+
+/// \file util/std_expr.h
+/// API to expression classes for bitvectors
+
+#include "std_expr.h"
```

Comment:
I understand that we need this right now, but could we consider adding something like `abstract_expr.h` that declares the likes of `unary_exprt` so that we can eventually speed up compile time a bit?

Reply:
Yes, that's the plan. I am considering to have `unary_exprt`, `binary_exprt` etc. in `expr.h`.